README.md

```diff
@@ -37,6 +74,6 @@ This PowerShell module downloads DSC Resources from the PSGallery and then build
 ### Building
 
 1. Install the EPS module from the PowerShell Gallery
-1. git clone jpogran/puppetdscbuilder
-1. Add DSC Resource names and versions to ./import.csv
-1. ./builder.ps1 -modulename 'dsc_foo'
+2. git clone jpogran/puppetdscbuilder
```

Comment:
I think this last chunk of changes can be skipped unless they came from linting?

This section is going to need a rewrite in the near future too, looks like - we don't use EPS or point to James' repo directly anymore.